apps/expo/src/app/userprofile/index.tsx

```diff
@@ -5,16 +5,50 @@ import { SafeAreaView } from "react-native-safe-area-context";
 import MIcon from "react-native-vector-icons/MaterialCommunityIcons";
 import { LinearGradient } from "expo-linear-gradient";
 
+import HorizontalList from "../../commons/HorizontalList";
 //import { MaterialCommunityIcons } from "react-native-vector-icons";
 import TitleStack from "../../commons/TitleStack";
 
+const defaultArray = [
```

Comment:
best approach is to set a TODO comment so we can know later on to change the hardcoded values